experiments/snippets/v2DragDrop/README.md

```diff
@@ -0,0 +1,133 @@
+# Drag and Drop in V2 Pages
+
+<img src="image3.gif" width="1000"></img>
+
+
+We’re exploring how to add “drag and drop” functionality to a Skuid page using a popular third-party JS library, [Dragula](https://bevacqua.github.io/dragula/). This light weight JS library has over 21k stars on Github and doesn’t use jQuery or other bloated frameworks. 
+
+Once the library is added, the Skuid builder can write JS to turn any data displaying components (Table, Deck, List) in Skuid into “drag and drop” mode. 
+
+## Install this example
+- Page API:  V2
+- Data source: Uses default Salesforce data source  (Cases)
+- Design system: Download and import [ProductTour_Kanban](ProductTour_Kanban.designsystem?raw=true)
+- Page XML:  [Copy the XML from this page](kanban_dragula.xml?raw=true), or save it as an XML file, and upload it.
+
+
+## Approach
```

Comment:
This is probably something to do across more experiments, but could we add a line here explaining that these steps walk you through what's already configured in the example above? Something to indicate "Hey if you've already installed the example, you can go ahead and look through it—or follow these steps to learn more" Maybe something like:

> Follow the instructions below to learn how the example above was built.

What you think?

Reply:
This makes sense.  I'll push up another commit in that line